dd-library-php-setup.php

```diff
@@ -189,6 +191,9 @@ function install($options)
                     "sed -i 's@extension \?= \?\(.*\)@extension = ddtrace.so@g' " . escapeshellarg($iniFilePath)
                 );
             }
+
+            add_missing_ini_settings($iniFilePath, get_ini_settings($installDirWrapperPath));
```

Comment:
Seems to me like there should be some error handling here, like if ini location doesn't exist or isn't reachable or something?

Reply:
In case anything goes wrong while writing the file, `add_missing_ini_settings` already prints an error and exist. The text is  `print_error_and_exit("Cannot add additional settings to the INI file $iniFilePath");`.

If you have something else in mind feel free to let me know and I can implement.